cves/CVE-2011-2729.yml

```diff
@@ -10,27 +10,27 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-284 Improper Access Control
 
 curated_instructions: |
   If you are manually editing this file, then you are "curating" it. Set the
   entry below to "true" as soon as you start. This will enable additional
   integrity checks on this file to make sure you fill everything out properly.
   If you are a student, we cannot accept your work as finished unless curated is
   set to true.
-curated: false
+curated: true
 
 reported_instructions: |
   Was there a date that this vulnerability was reported to the team? You can
   find this in changelogs, blogs, bug reports, or perhaps the CVE data.
   Please enter your date in YYYY-MM-DD format.
-reported:
+reported: 2011-09-20
```

Comment:
The date the vulnerability was reported to the team shouldn't be after it was announced to the world (the next section). Maybe you reversed these dates?

Reply:
The contest date section shows 2011-09-20, so it is the *announced* section that requires re-investigation.

Reply:
I accidentally put the date in the bounty section when it shouldn't have been there I will remove the date. I also fixed the reported and announced dates.